Build From Source error

In https://github.com/ciscospark/react-ciscospark/blob/master/README.md page, when I tried to install the build from source, after I ran the command “npm run build”, I met the following error. It seems it might be more related to the package itself. Please help give some advice on it, thanks in advance!

- KAYALI-M-Q4T0:react-ciscospark joyin$ npm run build

> @ciscospark/react-ciscospark@0.1.73 build /Applications/react-ciscospark
> npm run build:packages && npm run build:bundle


> @ciscospark/react-ciscospark@0.1.73 build:packages /Applications/react-ciscospark
> cross-env NODE_ENV=${NODE_ENV:-production} babel-node --presets=stage-1,es2015 scripts/build-packages.js

Cleaning react-component-activity-item dist folder...
Building react-component-activity-item...
Hash: 5d1986700df8e10adfaa
Version: webpack 2.6.1
Time: 16383ms
                        Asset     Size  Chunks                    Chunk Names
fonts/Atlantic-Neo-Icons.woff   104 kB          [emitted]         
                    bundle.js   439 kB       0  [emitted]  [big]  main
                     main.css  43.3 kB       0  [emitted]         main
                bundle.js.map  3.16 MB       0  [emitted]         main
                 main.css.map  15.3 kB       0  [emitted]         main
    + 529 hidden modules

WARNING in EnvironmentPlugin - MESSAGE_DEMO_CLIENT_ID environment variable is undefined.

You can pass an object with default values to suppress this warning.
See https://webpack.js.org/plugins/environment-plugin for example.

WARNING in EnvironmentPlugin - MESSAGE_DEMO_CLIENT_SECRET environment variable is undefined.

You can pass an object with default values to suppress this warning.
See https://webpack.js.org/plugins/environment-plugin for example.
react-component-activity-item... Done


Cleaning react-component-activity-item-base dist folder...
Building react-component-activity-item-base...
Hash: 84f4cac9de83bab0c93d
Version: webpack 2.6.1
Time: 4656ms
                        Asset     Size  Chunks             Chunk Names
fonts/Atlantic-Neo-Icons.woff   104 kB          [emitted]  
                    bundle.js    89 kB       0  [emitted]  main
                     main.css  38.9 kB       0  [emitted]  main
                bundle.js.map   476 kB       0  [emitted]  main
                 main.css.map  14.9 kB       0  [emitted]  main
    + 41 hidden modules

WARNING in EnvironmentPlugin - MESSAGE_DEMO_CLIENT_ID environment variable is undefined.

You can pass an object with default values to suppress this warning.
See https://webpack.js.org/plugins/environment-plugin for example.

WARNING in EnvironmentPlugin - MESSAGE_DEMO_CLIENT_SECRET environment variable is undefined.

You can pass an object with default values to suppress this warning.
See https://webpack.js.org/plugins/environment-plugin for example.
react-component-activity-item-base... Done


Cleaning react-component-activity-list dist folder...
Building react-component-activity-list...
Hash: a637551a944b2e1d79ab
Version: webpack 2.6.1
Time: 15001ms
                        Asset     Size  Chunks                    Chunk Names
fonts/Atlantic-Neo-Icons.woff   104 kB          [emitted]         
                    bundle.js   443 kB       0  [emitted]  [big]  main
                     main.css  44.5 kB       0  [emitted]         main
                bundle.js.map  3.19 MB       0  [emitted]         main
                 main.css.map  15.6 kB       0  [emitted]         main
    + 535 hidden modules

WARNING in EnvironmentPlugin - MESSAGE_DEMO_CLIENT_ID environment variable is undefined.

You can pass an object with default values to suppress this warning.
See https://webpack.js.org/plugins/environment-plugin for example.

WARNING in EnvironmentPlugin - MESSAGE_DEMO_CLIENT_SECRET environment variable is undefined.

You can pass an object with default values to suppress this warning.
See https://webpack.js.org/plugins/environment-plugin for example.
react-component-activity-list... Done


Cleaning react-component-activity-menu dist folder...
Building react-component-activity-menu...
Hash: 14db0b1ba4df1d7479ab
Version: webpack 2.6.1
Time: 4211ms
                        Asset     Size  Chunks             Chunk Names
fonts/Atlantic-Neo-Icons.woff   104 kB          [emitted]  
                    bundle.js  91.2 kB       0  [emitted]  main
                     main.css  35.8 kB       0  [emitted]  main
                bundle.js.map   506 kB       0  [emitted]  main
                 main.css.map  12.4 kB       0  [emitted]  main
    + 77 hidden modules

WARNING in EnvironmentPlugin - MESSAGE_DEMO_CLIENT_ID environment variable is undefined.

You can pass an object with default values to suppress this warning.
See https://webpack.js.org/plugins/environment-plugin for example.

WARNING in EnvironmentPlugin - MESSAGE_DEMO_CLIENT_SECRET environment variable is undefined.

You can pass an object with default values to suppress this warning.
See https://webpack.js.org/plugins/environment-plugin for example.
react-component-activity-menu... Done


Cleaning react-component-activity-post dist folder...
Building react-component-activity-post...
Hash: bf6bc544e44952ec60c1
Version: webpack 2.6.1
Time: 6090ms
                        Asset     Size  Chunks             Chunk Names
fonts/Atlantic-Neo-Icons.woff   104 kB          [emitted]  
                    bundle.js   142 kB       0  [emitted]  main
                     main.css  39.7 kB       0  [emitted]  main
                bundle.js.map  1.06 MB       0  [emitted]  main
                 main.css.map  14.2 kB       0  [emitted]  main
    + 295 hidden modules

WARNING in EnvironmentPlugin - MESSAGE_DEMO_CLIENT_ID environment variable is undefined.

You can pass an object with default values to suppress this warning.
See https://webpack.js.org/plugins/environment-plugin for example.

WARNING in EnvironmentPlugin - MESSAGE_DEMO_CLIENT_SECRET environment variable is undefined.

You can pass an object with default values to suppress this warning.
See https://webpack.js.org/plugins/environment-plugin for example.
react-component-activity-post... Done


Cleaning react-component-activity-post-action dist folder...
Building react-component-activity-post-action...
Hash: 537b9d38593ee0bf672a
Version: webpack 2.6.1
Time: 3725ms
                        Asset     Size  Chunks             Chunk Names
fonts/Atlantic-Neo-Icons.woff   104 kB          [emitted]  
                    bundle.js  83.6 kB       0  [emitted]  main
                     main.css  34.9 kB       0  [emitted]  main
                bundle.js.map   452 kB       0  [emitted]  main
                 main.css.map  11.5 kB       0  [emitted]  main
    + 37 hidden modules

WARNING in EnvironmentPlugin - MESSAGE_DEMO_CLIENT_ID environment variable is undefined.

You can pass an object with default values to suppress this warning.
See https://webpack.js.org/plugins/environment-plugin for example.

WARNING in EnvironmentPlugin - MESSAGE_DEMO_CLIENT_SECRET environment variable is undefined.

You can pass an object with default values to suppress this warning.
See https://webpack.js.org/plugins/environment-plugin for example.
react-component-activity-post-action... Done


Cleaning react-component-activity-share-file dist folder...
Building react-component-activity-share-file...
Hash: 0cf662b40eb0f2082785
Version: webpack 2.6.1
Time: 8898ms
                        Asset     Size  Chunks                    Chunk Names
fonts/Atlantic-Neo-Icons.woff   104 kB          [emitted]         
                    bundle.js   271 kB       0  [emitted]  [big]  main
                     main.css  35.7 kB       0  [emitted]         main
                bundle.js.map  1.86 MB       0  [emitted]         main
                 main.css.map  12.6 kB       0  [emitted]         main
    + 338 hidden modules

WARNING in EnvironmentPlugin - MESSAGE_DEMO_CLIENT_ID environment variable is undefined.

You can pass an object with default values to suppress this warning.
See https://webpack.js.org/plugins/environment-plugin for example.

WARNING in EnvironmentPlugin - MESSAGE_DEMO_CLIENT_SECRET environment variable is undefined.

You can pass an object with default values to suppress this warning.
See https://webpack.js.org/plugins/environment-plugin for example.
react-component-activity-share-file... Done


Cleaning react-component-activity-share-files dist folder...
Building react-component-activity-share-files...
Hash: f8fb9fb5658e43a40da2
Version: webpack 2.6.1
Time: 13327ms
                        Asset     Size  Chunks                    Chunk Names
fonts/Atlantic-Neo-Icons.woff   104 kB          [emitted]         
                    bundle.js   391 kB       0  [emitted]  [big]  main
                     main.css  42.9 kB       0  [emitted]         main
                bundle.js.map   2.8 MB       0  [emitted]         main
                 main.css.map  14.9 kB       0  [emitted]         main
    + 501 hidden modules

WARNING in EnvironmentPlugin - MESSAGE_DEMO_CLIENT_ID environment variable is undefined.

You can pass an object with default values to suppress this warning.
See https://webpack.js.org/plugins/environment-plugin for example.

WARNING in EnvironmentPlugin - MESSAGE_DEMO_CLIENT_SECRET environment variable is undefined.

You can pass an object with default values to suppress this warning.
See https://webpack.js.org/plugins/environment-plugin for example.
react-component-activity-share-files... Done


Cleaning react-component-activity-share-thumbnail dist folder...
Building react-component-activity-share-thumbnail...
Hash: e726bd027d32d0fb5b52
Version: webpack 2.6.1
Time: 8911ms
                        Asset     Size  Chunks                    Chunk Names
fonts/Atlantic-Neo-Icons.woff   104 kB          [emitted]         
                    bundle.js   273 kB       0  [emitted]  [big]  main
                     main.css  36.9 kB       0  [emitted]         main
                bundle.js.map  1.87 MB       0  [emitted]         main
                 main.css.map  13.5 kB       0  [emitted]         main
    + 343 hidden modules

WARNING in EnvironmentPlugin - MESSAGE_DEMO_CLIENT_ID environment variable is undefined.

You can pass an object with default values to suppress this warning.
See https://webpack.js.org/plugins/environment-plugin for example.

WARNING in EnvironmentPlugin - MESSAGE_DEMO_CLIENT_SECRET environment variable is undefined.

You can pass an object with default values to suppress this warning.
See https://webpack.js.org/plugins/environment-plugin for example.
react-component-activity-share-thumbnail... Done


Cleaning react-component-activity-system-message dist folder...
Building react-component-activity-system-message...
Hash: 3aa7c290d565d7c9464c
Version: webpack 2.6.1
Time: 4387ms
        Asset       Size  Chunks             Chunk Names
    bundle.js    73.4 kB       0  [emitted]  main
     main.css  201 bytes       0  [emitted]  main
bundle.js.map     682 kB       0  [emitted]  main
 main.css.map  368 bytes       0  [emitted]  main
    + 97 hidden modules

WARNING in EnvironmentPlugin - MESSAGE_DEMO_CLIENT_ID environment variable is undefined.

You can pass an object with default values to suppress this warning.
See https://webpack.js.org/plugins/environment-plugin for example.

WARNING in EnvironmentPlugin - MESSAGE_DEMO_CLIENT_SECRET environment variable is undefined.

You can pass an object with default values to suppress this warning.
See https://webpack.js.org/plugins/environment-plugin for example.
react-component-activity-system-message... Done


Cleaning react-component-activity-text dist folder...
Building react-component-activity-text...
Hash: 042a04aab5cc98269984
Version: webpack 2.6.1
Time: 4835ms
        Asset       Size  Chunks             Chunk Names
    bundle.js    74.1 kB       0  [emitted]  main
     main.css  947 bytes       0  [emitted]  main
bundle.js.map     854 kB       0  [emitted]  main
 main.css.map    1.32 kB       0  [emitted]  main
    + 284 hidden modules

WARNING in EnvironmentPlugin - MESSAGE_DEMO_CLIENT_ID environment variable is undefined.

You can pass an object with default values to suppress this warning.
See https://webpack.js.org/plugins/environment-plugin for example.

WARNING in EnvironmentPlugin - MESSAGE_DEMO_CLIENT_SECRET environment variable is undefined.

You can pass an object with default values to suppress this warning.
See https://webpack.js.org/plugins/environment-plugin for example.
react-component-activity-text... Done


Cleaning react-component-add-file-button dist folder...
Building react-component-add-file-button...
Hash: 18bb811a42e1bc97a5d4
Version: webpack 2.6.1
Time: 3557ms
                        Asset     Size  Chunks             Chunk Names
fonts/Atlantic-Neo-Icons.woff   104 kB          [emitted]  
                    bundle.js  82.8 kB       0  [emitted]  main
                     main.css  35.1 kB       0  [emitted]  main
                bundle.js.map   447 kB       0  [emitted]  main
                 main.css.map  12.2 kB       0  [emitted]  main
    + 35 hidden modules

WARNING in EnvironmentPlugin - MESSAGE_DEMO_CLIENT_ID environment variable is undefined.

You can pass an object with default values to suppress this warning.
See https://webpack.js.org/plugins/environment-plugin for example.

WARNING in EnvironmentPlugin - MESSAGE_DEMO_CLIENT_SECRET environment variable is undefined.

You can pass an object with default values to suppress this warning.
See https://webpack.js.org/plugins/environment-plugin for example.
react-component-add-file-button... Done


Cleaning react-component-avatar dist folder...
Building react-component-avatar...
Hash: b6250af5350d4069a8d3
Version: webpack 2.6.1
Time: 3639ms
                        Asset     Size  Chunks             Chunk Names
fonts/Atlantic-Neo-Icons.woff   104 kB          [emitted]  
                    bundle.js  83.4 kB       0  [emitted]  main
                     main.css  35.9 kB       0  [emitted]  main
                bundle.js.map   450 kB       0  [emitted]  main
                 main.css.map  12.8 kB       0  [emitted]  main
    + 35 hidden modules

WARNING in EnvironmentPlugin - MESSAGE_DEMO_CLIENT_ID environment variable is undefined.

You can pass an object with default values to suppress this warning.
See https://webpack.js.org/plugins/environment-plugin for example.

WARNING in EnvironmentPlugin - MESSAGE_DEMO_CLIENT_SECRET environment variable is undefined.

You can pass an object with default values to suppress this warning.
See https://webpack.js.org/plugins/environment-plugin for example.
react-component-avatar... Done


Cleaning react-component-button dist folder...
Building react-component-button...
Hash: 1eff207204f8ce1bc697
Version: webpack 2.6.1
Time: 3580ms
                        Asset     Size  Chunks             Chunk Names
fonts/Atlantic-Neo-Icons.woff   104 kB          [emitted]  
                    bundle.js    83 kB       0  [emitted]  main
                     main.css  34.8 kB       0  [emitted]  main
                bundle.js.map   448 kB       0  [emitted]  main
                 main.css.map  11.6 kB       0  [emitted]  main
    + 35 hidden modules

WARNING in EnvironmentPlugin - MESSAGE_DEMO_CLIENT_ID environment variable is undefined.

You can pass an object with default values to suppress this warning.
See https://webpack.js.org/plugins/environment-plugin for example.

WARNING in EnvironmentPlugin - MESSAGE_DEMO_CLIENT_SECRET environment variable is undefined.

You can pass an object with default values to suppress this warning.
See https://webpack.js.org/plugins/environment-plugin for example.
react-component-button... Done


Cleaning react-component-button-controls dist folder...
Building react-component-button-controls...
Hash: 5384aaeab2c8fdbcbbbf
Version: webpack 2.6.1
Time: 4151ms
                        Asset     Size  Chunks             Chunk Names
fonts/Atlantic-Neo-Icons.woff   104 kB          [emitted]  
                    bundle.js    90 kB       0  [emitted]  main
                     main.css  35.2 kB       0  [emitted]  main
                bundle.js.map   500 kB       0  [emitted]  main
                 main.css.map  11.9 kB       0  [emitted]  main
    + 75 hidden modules

WARNING in EnvironmentPlugin - MESSAGE_DEMO_CLIENT_ID environment variable is undefined.

You can pass an object with default values to suppress this warning.
See https://webpack.js.org/plugins/environment-plugin for example.

WARNING in EnvironmentPlugin - MESSAGE_DEMO_CLIENT_SECRET environment variable is undefined.

You can pass an object with default values to suppress this warning.
See https://webpack.js.org/plugins/environment-plugin for example.
react-component-button-controls... Done


Cleaning react-component-chip-base dist folder...
Building react-component-chip-base...
Hash: ec2414cc0e03f308d592
Version: webpack 2.6.1
Time: 3700ms
                        Asset     Size  Chunks             Chunk Names
fonts/Atlantic-Neo-Icons.woff   104 kB          [emitted]  
                    bundle.js  83.7 kB       0  [emitted]  main
                     main.css  35.4 kB       0  [emitted]  main
                bundle.js.map   453 kB       0  [emitted]  main
                 main.css.map  12.1 kB       0  [emitted]  main
    + 37 hidden modules

WARNING in EnvironmentPlugin - MESSAGE_DEMO_CLIENT_ID environment variable is undefined.

You can pass an object with default values to suppress this warning.
See https://webpack.js.org/plugins/environment-plugin for example.

WARNING in EnvironmentPlugin - MESSAGE_DEMO_CLIENT_SECRET environment variable is undefined.

You can pass an object with default values to suppress this warning.
See https://webpack.js.org/plugins/environment-plugin for example.
react-component-chip-base... Done


Cleaning react-component-chip-file dist folder...
Building react-component-chip-file...
Hash: 1764e9dc7ce801ee1d96
Version: webpack 2.6.1
Time: 3964ms
                        Asset     Size  Chunks             Chunk Names
fonts/Atlantic-Neo-Icons.woff   104 kB          [emitted]  
                    bundle.js  85.2 kB       0  [emitted]  main
                     main.css  36.5 kB       0  [emitted]  main
                bundle.js.map   460 kB       0  [emitted]  main
                 main.css.map  13.1 kB       0  [emitted]  main
    + 39 hidden modules

WARNING in EnvironmentPlugin - MESSAGE_DEMO_CLIENT_ID environment variable is undefined.

You can pass an object with default values to suppress this warning.
See https://webpack.js.org/plugins/environment-plugin for example.

WARNING in EnvironmentPlugin - MESSAGE_DEMO_CLIENT_SECRET environment variable is undefined.

You can pass an object with default values to suppress this warning.
See https://webpack.js.org/plugins/environment-plugin for example.
react-component-chip-file... Done


Cleaning react-component-confirmation-modal dist folder...
Building react-component-confirmation-modal...
Hash: 08161c08208cc5df0357
Version: webpack 2.6.1
Time: 2804ms
        Asset     Size  Chunks             Chunk Names
    bundle.js  26.1 kB       0  [emitted]  main
     main.css  4.61 kB       0  [emitted]  main
bundle.js.map   292 kB       0  [emitted]  main
 main.css.map  6.26 kB       0  [emitted]  main
    + 33 hidden modules

WARNING in EnvironmentPlugin - MESSAGE_DEMO_CLIENT_ID environment variable is undefined.

You can pass an object with default values to suppress this warning.
See https://webpack.js.org/plugins/environment-plugin for example.

WARNING in EnvironmentPlugin - MESSAGE_DEMO_CLIENT_SECRET environment variable is undefined.

You can pass an object with default values to suppress this warning.
See https://webpack.js.org/plugins/environment-plugin for example.
react-component-confirmation-modal... Done


Cleaning react-component-cover dist folder...
Building react-component-cover...
Hash: c03b38baa2686bbeb47e
Version: webpack 2.6.1
Time: 2666ms
        Asset       Size  Chunks             Chunk Names
    bundle.js    21.7 kB       0  [emitted]  main
     main.css  389 bytes       0  [emitted]  main
bundle.js.map     279 kB       0  [emitted]  main
 main.css.map  619 bytes       0  [emitted]  main
    + 33 hidden modules

WARNING in EnvironmentPlugin - MESSAGE_DEMO_CLIENT_ID environment variable is undefined.

You can pass an object with default values to suppress this warning.
See https://webpack.js.org/plugins/environment-plugin for example.

WARNING in EnvironmentPlugin - MESSAGE_DEMO_CLIENT_SECRET environment variable is undefined.

You can pass an object with default values to suppress this warning.
See https://webpack.js.org/plugins/environment-plugin for example.
react-component-cover... Done


Cleaning react-component-day-separator dist folder...
Building react-component-day-separator...
Hash: 09b129893109bfa6db76
Version: webpack 2.6.1
Time: 5232ms
        Asset     Size  Chunks             Chunk Names
    bundle.js   118 kB       0  [emitted]  main
     main.css  1.22 kB       0  [emitted]  main
bundle.js.map   977 kB       0  [emitted]  main
 main.css.map  1.64 kB       0  [emitted]  main
    + 61 hidden modules

WARNING in EnvironmentPlugin - MESSAGE_DEMO_CLIENT_ID environment variable is undefined.

You can pass an object with default values to suppress this warning.
See https://webpack.js.org/plugins/environment-plugin for example.

WARNING in EnvironmentPlugin - MESSAGE_DEMO_CLIENT_SECRET environment variable is undefined.

You can pass an object with default values to suppress this warning.
See https://webpack.js.org/plugins/environment-plugin for example.
react-component-day-separator... Done


Cleaning react-component-error-display dist folder...
Building react-component-error-display...
Hash: b0f732fde00eaa7e23f8
Version: webpack 2.6.1
Time: 2878ms
        Asset       Size  Chunks             Chunk Names
    bundle.js    26.7 kB       0  [emitted]  main
     main.css  967 bytes       0  [emitted]  main
bundle.js.map     315 kB       0  [emitted]  main
 main.css.map    1.23 kB       0  [emitted]  main
    + 54 hidden modules

WARNING in EnvironmentPlugin - MESSAGE_DEMO_CLIENT_ID environment variable is undefined.

You can pass an object with default values to suppress this warning.
See https://webpack.js.org/plugins/environment-plugin for example.

WARNING in EnvironmentPlugin - MESSAGE_DEMO_CLIENT_SECRET environment variable is undefined.

You can pass an object with default values to suppress this warning.
See https://webpack.js.org/plugins/environment-plugin for example.
react-component-error-display... Done


Cleaning react-component-file-staging-area dist folder...
Building react-component-file-staging-area...
Hash: 62df18ccb7d3a76caad9
Version: webpack 2.6.1
Time: 9147ms
                        Asset     Size  Chunks                    Chunk Names
fonts/Atlantic-Neo-Icons.woff   104 kB          [emitted]         
                    bundle.js   272 kB       0  [emitted]  [big]  main
                     main.css  37.3 kB       0  [emitted]         main
                bundle.js.map  1.87 MB       0  [emitted]         main
                 main.css.map  13.3 kB       0  [emitted]         main
    + 342 hidden modules

WARNING in EnvironmentPlugin - MESSAGE_DEMO_CLIENT_ID environment variable is undefined.

You can pass an object with default values to suppress this warning.
See https://webpack.js.org/plugins/environment-plugin for example.

WARNING in EnvironmentPlugin - MESSAGE_DEMO_CLIENT_SECRET environment variable is undefined.

You can pass an object with default values to suppress this warning.
See https://webpack.js.org/plugins/environment-plugin for example.
react-component-file-staging-area... Done


Cleaning react-component-icon dist folder...
Building react-component-icon...
Hash: 3ecb927c56588eef9358
Version: webpack 2.6.1
Time: 3458ms
                        Asset     Size  Chunks             Chunk Names
fonts/Atlantic-Neo-Icons.woff   104 kB          [emitted]  
                    bundle.js  81.8 kB       0  [emitted]  main
                     main.css  34.6 kB       0  [emitted]  main
                bundle.js.map   442 kB       0  [emitted]  main
                 main.css.map  39.9 kB       0  [emitted]  main
    + 33 hidden modules

WARNING in EnvironmentPlugin - MESSAGE_DEMO_CLIENT_ID environment variable is undefined.

You can pass an object with default values to suppress this warning.
See https://webpack.js.org/plugins/environment-plugin for example.

WARNING in EnvironmentPlugin - MESSAGE_DEMO_CLIENT_SECRET environment variable is undefined.

You can pass an object with default values to suppress this warning.
See https://webpack.js.org/plugins/environment-plugin for example.
react-component-icon... Done


Cleaning react-component-incoming-call dist folder...
Building react-component-incoming-call...
Hash: 88fd5df579c43e290907
Version: webpack 2.6.1
Time: 4318ms
                        Asset     Size  Chunks             Chunk Names
fonts/Atlantic-Neo-Icons.woff   104 kB          [emitted]  
                    bundle.js  93.8 kB       0  [emitted]  main
                     main.css  37.5 kB       0  [emitted]  main
                bundle.js.map   517 kB       0  [emitted]  main
                 main.css.map  13.2 kB       0  [emitted]  main
    + 79 hidden modules

WARNING in EnvironmentPlugin - MESSAGE_DEMO_CLIENT_ID environment variable is undefined.

You can pass an object with default values to suppress this warning.
See https://webpack.js.org/plugins/environment-plugin for example.

WARNING in EnvironmentPlugin - MESSAGE_DEMO_CLIENT_SECRET environment variable is undefined.

You can pass an object with default values to suppress this warning.
See https://webpack.js.org/plugins/environment-plugin for example.
react-component-incoming-call... Done


Cleaning react-component-list-separator dist folder...
Building react-component-list-separator...
Hash: 94f647a74849f2559d60
Version: webpack 2.6.1
Time: 2700ms
        Asset     Size  Chunks             Chunk Names
    bundle.js    22 kB       0  [emitted]  main
     main.css  1.22 kB       0  [emitted]  main
bundle.js.map   280 kB       0  [emitted]  main
 main.css.map  1.64 kB       0  [emitted]  main
    + 33 hidden modules

WARNING in EnvironmentPlugin - MESSAGE_DEMO_CLIENT_ID environment variable is undefined.

You can pass an object with default values to suppress this warning.
See https://webpack.js.org/plugins/environment-plugin for example.

WARNING in EnvironmentPlugin - MESSAGE_DEMO_CLIENT_SECRET environment variable is undefined.

You can pass an object with default values to suppress this warning.
See https://webpack.js.org/plugins/environment-plugin for example.
react-component-list-separator... Done


Cleaning react-component-loading-screen dist folder...
Building react-component-loading-screen...
Hash: e6fefce052d1560b4dcf
Version: webpack 2.6.1
Time: 3261ms
        Asset       Size  Chunks             Chunk Names
    bundle.js    27.3 kB       0  [emitted]  main
     main.css  670 bytes       0  [emitted]  main
bundle.js.map     316 kB       0  [emitted]  main
 main.css.map  938 bytes       0  [emitted]  main
    + 56 hidden modules

WARNING in EnvironmentPlugin - MESSAGE_DEMO_CLIENT_ID environment variable is undefined.

You can pass an object with default values to suppress this warning.
See https://webpack.js.org/plugins/environment-plugin for example.

WARNING in EnvironmentPlugin - MESSAGE_DEMO_CLIENT_SECRET environment variable is undefined.

You can pass an object with default values to suppress this warning.
See https://webpack.js.org/plugins/environment-plugin for example.

ERROR in ./styles.css
Module build failed: ModuleBuildError: Module build failed: Error: spawn /Applications/react-ciscospark/node_modules/pngquant-bin/vendor/pngquant ENOENT
    at exports._errnoException (util.js:1050:11)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:193:32)
    at onErrorNT (internal/child_process.js:367:16)
    at _combinedTickCallback (internal/process/next_tick.js:80:11)
    at process._tickCallback (internal/process/next_tick.js:104:9)
    at runLoaders (/Applications/react-ciscospark/node_modules/webpack/lib/NormalModule.js:192:19)
    at /Applications/react-ciscospark/node_modules/loader-runner/lib/LoaderRunner.js:364:11
    at /Applications/react-ciscospark/node_modules/loader-runner/lib/LoaderRunner.js:230:18
    at context.callback (/Applications/react-ciscospark/node_modules/loader-runner/lib/LoaderRunner.js:111:13)
    at imagemin.buffer.then.catch.err (/Applications/react-ciscospark/node_modules/image-webpack-loader/index.js:80:9)
 @ ./index.js 20:14-37

ERROR in ./styles.css
Module build failed: ModuleBuildError: Module build failed: Error: spawn /Applications/react-ciscospark/node_modules/pngquant-bin/vendor/pngquant ENOENT
    at exports._errnoException (util.js:1050:11)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:193:32)
    at onErrorNT (internal/child_process.js:367:16)
    at _combinedTickCallback (internal/process/next_tick.js:80:11)
    at process._tickCallback (internal/process/next_tick.js:104:9)
    at runLoaders (/Applications/react-ciscospark/node_modules/webpack/lib/NormalModule.js:192:19)
    at /Applications/react-ciscospark/node_modules/loader-runner/lib/LoaderRunner.js:364:11
    at /Applications/react-ciscospark/node_modules/loader-runner/lib/LoaderRunner.js:230:18
    at context.callback (/Applications/react-ciscospark/node_modules/loader-runner/lib/LoaderRunner.js:111:13)
    at imagemin.buffer.then.catch.err (/Applications/react-ciscospark/node_modules/image-webpack-loader/index.js:80:9)

ERROR in /Applications/react-ciscospark/node_modules/extract-text-webpack-plugin/loader.js??ref--1-0!/Applications/react-ciscospark/node_modules/css-loader/index.js??ref--1-1!/Applications/react-ciscospark/node_modules/postcss-loader/lib/index.js??ref--1-2!/Applications/react-ciscospark/packages/node_modules/@ciscospark/react-component-loading-screen/src/styles.css doesn't export content
/Applications/react-ciscospark/scripts/build-packages.js:21
    throw err;
    ^

Error: Error building react-component-loading-screen package, Error: Command failed: cd /Applications/react-ciscospark/packages/node_modules/@ciscospark/react-component-loading-screen && webpack --config /Applications/react-ciscospark/scripts/webpack/webpack.prod.babel.js
    at buildPackage (/Applications/react-ciscospark/scripts/build-package.js:25:13)
    at /Applications/react-ciscospark/scripts/build-packages.js:9:12
    at Array.map (native)
    at Object.<anonymous> (/Applications/react-ciscospark/scripts/build-packages.js:6:22)
    at Module._compile (module.js:571:32)
    at loader (/Applications/react-ciscospark/node_modules/babel-register/lib/node.js:144:5)
    at Object.require.extensions.(anonymous function) [as .js] (/Applications/react-ciscospark/node_modules/babel-register/lib/node.js:154:7)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @ciscospark/react-ciscospark@0.1.73 build:packages: `cross-env NODE_ENV=${NODE_ENV:-production} babel-node --presets=stage-1,es2015 scripts/build-packages.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @ciscospark/react-ciscospark@0.1.73 build:packages script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/joyin/.npm/_logs/2017-05-30T08_49_54_590Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @ciscospark/react-ciscospark@0.1.73 build: `npm run build:packages && npm run build:bundle`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @ciscospark/react-ciscospark@0.1.73 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/joyin/.npm/_logs/2017-05-30T08_49_54_635Z-debug.log
KAYALI-M-Q4T0:react-ciscospark joyin$

--- a/package.json
+++ b/package.json
@@ -120,7 +120,6 @@
     "extract-text-webpack-plugin": "^2.0.0",
     "file-loader": "^0.11.1",
     "html-webpack-plugin": "^2.24.1",
-    "image-webpack-loader": "^3.1.0",
     "inline-environment-variables-webpack-plugin": "^1.1.0",
     "jest": "^20.0.4",
     "json-loader": "^0.5.4",

--- a/scripts/webpack/webpack.base.babel.js
+++ b/scripts/webpack/webpack.base.babel.js
@@ -121,8 +121,7 @@ export default (options) => {
         {
           test: /.*\.(gif|png|jpg)$/,
           use: [
-            `file-loader?name=[name].[ext]`,
-            `image-webpack-loader?{optimizationLevel: 7, interlaced: false, pngquant:{quality: "65-90", speed: 4}, mozjpeg: {quality: 65}}`
+            `file-loader?name=[name].[ext]`
           ]
         }
       ]

